x/profile/keeper/msg_server_disable_coordinator.go

```diff
@@ -2,36 +2,37 @@ package keeper
 
 import (
 	"context"
-
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	spnerrors "github.com/tendermint/spn/pkg/errors"
 	"github.com/tendermint/spn/x/profile/types"
 )
 
-func (k msgServer) DeleteCoordinator(
+func (k msgServer) DisableCoordinator(
 	goCtx context.Context,
-	msg *types.MsgDeleteCoordinator,
-) (*types.MsgDeleteCoordinatorResponse, error) {
+	msg *types.MsgDisableCoordinator,
+) (*types.MsgDisableCoordinatorResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
 	// Check if the coordinator address is already in the store
 	coordByAddress, found := k.GetCoordinatorByAddress(ctx, msg.Address)
 	if !found {
-		return &types.MsgDeleteCoordinatorResponse{},
+		return &types.MsgDisableCoordinatorResponse{},
 			sdkerrors.Wrap(types.ErrCoordAddressNotFound, msg.Address)
 	}
 
 	coord, found := k.GetCoordinator(ctx, coordByAddress.CoordinatorID)
 	if !found {
-		return &types.MsgDeleteCoordinatorResponse{},
+		return &types.MsgDisableCoordinatorResponse{},
 			spnerrors.Criticalf("a coordinator address is associated to a non-existent coordinator ID: %d",
 				coordByAddress.CoordinatorID)
 	}
 
-	k.RemoveCoordinatorByAddress(ctx, msg.Address)
```

Comment:
We can keep removing this particular structure

The address will remain documented since this is part of `Coordinator` structure
This would allow the user to create a new coordinator